logdetective/server/models.py

```diff
@@ -37,24 +37,29 @@ class JobHook(BaseModel):
     project_id: int
 
 
+class Explanation(BaseModel):
+    """Model of snippet or general log explanation from Log Detective"""
+
+    text: str
+    logprobs: Optional[List[Dict]] = None
+
+
```

Comment:
Can we simplify the API for the snippets below by making Explanation behave like a `str` when needed?
```suggestion
class Explanation(BaseModel):
    """Model of snippet or general log explanation from Log Detective"""

    text: str
    logprobs: Optional[List[Dict]] = None

    def __str__(self):
        return self.text
```

Reply:
I get what you mean. I'm not sure how useful it will be in practice but I'll include it. The whole response will have to be redone.

Reply:
It just seems like it'll be easier on the consumer, rather than them needing to interrogate the returned object to see which type they received. We always return an `Explanation` and we can coerce it into a `str` if an existing client (are there any?) needs it to behave like a string.